src/java/org/apache/cassandra/db/marshal/AbstractType.java

```diff
@@ -415,6 +415,11 @@ public boolean isTuple()
         return false;
     }
 
+    public boolean isTruncatable()
+    {
+        return false;
+    }
+
```

Comment:
The truncation / rounding behavior is SAI specific. This shouldn't be in `db.marshal.AbstractType`, but rather in `TypeUtil` maybe?

Reply:
Thank you for the suggestion. Actually there is already function doing what I need: `supportsRounding`. So I will just use it.